test/formatters/installers/ktlint.bash

```diff
@@ -0,0 +1,4 @@
+cd /usr/local/bin
+ver="$(latest_release pinterest/ktlint)"
+apt install -y default-jre
```

Comment:
```suggestion
apt-get install -y default-jre
```

Reply:
`apt-get` is generally recommended for programmatic usage and scripts while `apt` is a wrapper intended to provide more convenient command-line usage for humans.